test/endtoend/providers/test.js

```diff
@@ -10,24 +51,36 @@ describe('File upload with Providers', () => {
     browser.reload()
   })
 
-  it('should upload a file completely with Google Drive', () => {
+  it('should upload a file completely with Google Drive', function () {
```

Comment:
why are we not using arrow functions in this case?

Reply:
So that it gets mocha's `this` for the `this.skip` call below

Reply:
yeah, maybe it'll be better to add a comment right above it, otherwise one (e.g Me) might wrongfully change it in the future 🙂